ci-operator/step-registry/ipi/conf/vsphere/zones/ipi-conf-vsphere-zones-commands.sh

```diff
@@ -24,6 +24,8 @@ mapfile -t vips < "${SHARED_DIR}/vips.txt"
 CONFIG="${SHARED_DIR}/install-config.yaml"
 base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
 machine_cidr=$(<"${SHARED_DIR}"/machinecidr.txt)
+ova_url=$(openshift-install coreos print-stream-json | jq -r '.architectures.x86_64.artifacts.vmware.formats.ova.disk.location')
+vm_template="${ova_url##*/}"
```

Comment:
As I know, the output of `openshift-install coreos print-stream-json` may change from version to version, so suggest to add a validation check on the value of `ova_url` here to make sure it's a valid one.

Reply:
nevermind, if the output changes, the `jq` command should fail directly and the script aborted.